logos/logos-orchestrator/src/logos/pipeline/executor.py

```
@@ -90,14 +97,19 @@ async def execute_streaming(
 
                 try:
-                    async for line in resp.aiter_lines():
-                        if line:
-                            yield (line + "\n").encode()
+                    # Preserve upstream byte framing. SSE uses blank lines to
+                    # delimit events, and local providers may stream NDJSON;
+                    # reconstructing either format line-by-line changes it.
+                    async for chunk in resp.aiter_bytes():
+                        if chunk:
+                            yield chunk
                 except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)